zokrates_bellman_plonk/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "zokrates_bellman_plonk"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+wasm = ["bellman_ce/nolog", "bellman_ce/wasm"]
+multicore = ["bellman_ce/multicore", "phase2/multicore"]
+
+[dependencies]
+zokrates_field = { version = "0.5", path = "../zokrates_field", default-features = false }
+zokrates_ast = { version = "0.1", path = "../zokrates_ast", default-features = false }
+zokrates_proof_systems = { version = "0.1", path = "../zokrates_proof_systems", default-features = false }
+
+bellman_ce = { git = "https://github.com/georgwiese/bellman", rev="b356c7001f30da23bfad2b43eb0b7ca9804c8252", default-features = false, features = ["plonk", "multicore"] }
```

Comment:
Shall we try to get these changes merged?

Reply:
Sure! I opened https://github.com/matter-labs/bellman/pull/48. Also, I pulled the most recent `dev` and updated our `Cargo.toml`s accordingly to make sure that we can actually use the merged version.